index.js

```diff
@@ -165,6 +166,8 @@ We suggest that you begin by typing:`)
     {bold ${runCommand} dev}`)
   }
 
+  mixpanel.track(frontend, contract)
```

Comment:
@icerove I would not put `NEAR collect the type of UI and contract...` info at the end of the build. We really want developers to concentrate on `cd appName` and `yarn dev` commands. I would rather put it at the beginning of the function.